config/application.rb

```diff
@@ -31,5 +31,7 @@ class Application < Rails::Application
     config.browserify_rails.commandline_options = '-t [ babelify --presets [ es2015 ] ]'
 
     config.i18n.load_path += Dir[Rails.root.join('config', 'locales', '**', '*.{yml}')]
+
+    routes.default_url_options[:host] = Figaro.env.mailer_domain_name
```

Comment:
Note that the `host` option in Rails configuration never includes the protocol. The [protocol can be specified](https://github.com/18F/identity-idp/blob/master/config/environments/development.rb#L15) separately if desired.